management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java

```diff
@@ -241,7 +242,7 @@ public void test_stats_collection() throws Exception {
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
     assertThat(stats[0].getStatistic(Counter.class, "GetCounter").getValue(), equalTo(4L));
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
-    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, equalTo(1));
+    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, greaterThanOrEqualTo(1));
```

Comment:
In master:

- __T__ : start collector scheduler
- __get operations__
- __T + 1 sec__ : collector scheduler:
  - collects stats (__=> sample computation starts now__)
  - send them to voltron
- __T + 1.5 sec__ : [waitForNextStats](https://github.com/anthonydahanne/ehcache3/blob/master/management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java#L224)
  - we have 1 stat and __no sample__
- __get operations__
- __T + 2 sec__ : collector scheduler runs
  - collects stats
  - send them to voltron
- __T + 3 sec__ : collector scheduler runs
  - collects stats
  - send them to voltron
- __T + 3 sec__ : [waitForNextStats] (https://github.com/anthonydahanne/ehcache3/blob/master/management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java#L239)
  - we have 1 stat and __1 or 2 sample values, depending whether the collector ran after or before this__ Values will all be 4.

In PR #1390:

- __T__ : start collector scheduler __AND sample computation starts now__
- __get operations__
- __T + 1 sec__ : collector scheduler:
  - collects stats
  - send them to voltron
- __T + 1.5 sec__ : [waitForNextStats](https://github.com/anthonydahanne/ehcache3/blob/master/management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java#L224)
  - we have 1 stat and __1 sample value that is 2__
- __get operations__
- __T + 2 sec__ : collector scheduler runs
  - collects stats
  - send them to voltron
- __T + 3 sec__ : collector scheduler runs
  - collects stats
  - send them to voltron
- __T + 3 sec__ : [waitForNextStats] (https://github.com/anthonydahanne/ehcache3/blob/master/management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java#L239)
  - we have 1 stat and __2 or 3 sample values, depending whether the collector ran after or before this__ Values will be 2, 4 OR 2, 4, 4